.github/workflows/release.yml

```
@@ -67,6 +67,8 @@ jobs:
           ARTIFACTORY_USER: ${{ secrets.ARTIFACTORY_USER }}
           CIRCLE_TOKEN: ${{ secrets.CIRCLE_TOKEN }}
           SIGNORE_SIGNER: ${{secrets.SIGNORE_SIGNER}}
+          SIGNORE_CLIENT_ID: $${{secrets.SIGNORE_CLIENT_ID}
+          SIGNORE_CLIENT_SECRET: $${{secrets.SIGNORE_CLIENT_SECRET}
```

Extra `$` and missing `}` 😎 

```suggestion
          SIGNORE_CLIENT_ID: ${{secrets.SIGNORE_CLIENT_ID}}
          SIGNORE_CLIENT_SECRET: ${{secrets.SIGNORE_CLIENT_SECRET}}
```

---

Oh geez, thanks for catching this Brian. :D